### PR TITLE
Adding hackeve_47

### PR DESCRIPTION
**Pull request by amehra-github**
Information regarding hackeve 47

### CODE REVIEW

**Comment by peey** [comment]
Add @suryatejreddy as admin-poc. @suryatejreddy please proofread and merge after the changes

**Comment by amehra-github** [comment]
Done

**Comment by amehra-github** [comment]
Made the changes

File: current/hackeve47_intro_to_pygame [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+type: Hackeve 47
```
Comment (sedflix) [inline current/hackeve47_intro_to_pygame]: It should be just "HackEve". 
You can make the name as HackEve 45: Intro to pygame.

**Comment by peey** [comment]
Assuming everything is correct. Closes https://github.com/IIIT-Delhi/talks/issues/17

**Pull request merged**